horovod/run/run.py

```diff
@@ -427,8 +430,10 @@ def parse_args():
                              'HOROVOD_START_TIMEOUT can also be used to '
                              'specify the initialization timeout.')
 
-    parser.add_argument('--network-interface', action='store', dest='nic',
-                        help='Specify the network interface used for communication.')
+    parser.add_argument('--network-interfaces', action='store', dest='nics',
```

Comment:
Let's avoid changing the name of this parameter.  Some users are already using it, and we don't want to suddenly break them.  I think it's okay if it's called "nic" and it accepts a comma-separated list.

Reply:
changed.